### ISSUE TITLE
Should not check module name in require lines.

### ISSUE THREAD

**Issue by xpol** [Jason-Rev/vscode-spell-checker#314]
## What

```js
import shouldCheck from "shouldNotCheck"
const shouldCheckToo = require("shouldNotCheckToo")
```
The `shouldNotCheck` and `shouldNotCheckToo` should not checked by vscode-spell-checker.

## Why

Package names may be not have valid spells. eg `antd`

But they may widely used in whole project. To disable them manually is not necessary.

**Comment by Jason3S** [comment]
I fully agree with you that it should not spell check imports.

I am working on a way to do this, but it will take some time.

**Comment by OmgImAlexis** [comment]
@Jason3S is there a regex we could use for now until a proper workaround is added?

**Comment by OmgImAlexis** [comment]
I'm thinking something like this might work?

```js
// cSpell:ignoreRegExp (?:from (?:'|")([A-Za-z0-9-_\/\.\@]*)(?:'|"))|(?:require\((?:'|")([A-Za-z0-9-_\/\.\@]*)(?:'|")\)) -- ignore node's import/require
import shouldCheck from "shouldNotCheck"
const shouldCheckToo = require("shouldNotCheckToo")
```
